`make  'forced-clean-sql'` fails with Docker Toolbox for Windows

#### The problem
`./quickstart.sh` fails with the following error:
```
====> : Drop and Recreate PostgreSQL  public schema
docker-compose run --rm import-osm /usr/src/app/psql.sh -c "DROP SCHEMA IF EXISTS public CASCADE ; CREATE SCHEMA IF NOT EXISTS public; "
Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"C:/Program Files/Git/usr/src/app/psql.sh\": stat C:/Program Files/Git/usr/src/app/psql.sh: no such file or directory": unknown
Makefile:192: recipe for target 'forced-clean-sql' failed
make: *** [forced-clean-sql] Error 1
```
#### Environment
Windows 10 Home
Docker Toolbox for Windows 18.03.0-ce, build 0520e24302
docker-compose version 1.20.1, build 5d8c71b2
openmaptiles/import-osm 0.5

#### Analysis
Docker Toolbox for Windows adds a `C:/Program Files/Git` prefix to the `/usr/src/app/psql.sh` command. This is a Toolbox for Windows bug.

However, this bug does not manifest itself in earlier `./quickstart.sh` use of `docker-compose run --rm import-osm` in `Makefile` targets, such as the `make download-geofabrik area=albania`:
```
docker-compose run --rm import-osm  ./download-geofabrik.sh albania
```
This is likely because `./download-geofabrik.sh` starts with a `.`.

#### Proposed solution
Since
```
$ docker-compose run --rm import-osm pwd
/usr/src/app
```
All references to `docker-compose run --rm import-osm /usr/src/app/...` in `Makefile` can be safely changed to `docker-compose run --rm import-osm ./...`, just as it is already done for downloading the OSM data.

--- a/Makefile
+++ b/Makefile
@@ -68,7 +68,7 @@ download-geofabrik:
 	@echo " "
 
 psql: db-start
-	docker-compose run --rm import-osm /usr/src/app/psql.sh
+	docker-compose run --rm import-osm ./psql.sh
 
 cfg-remake:
 	docker-compose run --rm openmaptiles-tools make clean
@@ -183,29 +183,29 @@ download-wikidata:
 	mkdir -p wikidata && docker-compose run --rm --entrypoint /usr/src/app/download-gz.sh import-wikidata
 
 psql-list-tables:
-	docker-compose run --rm import-osm /usr/src/app/psql.sh  -P pager=off  -c "\d+"
+	docker-compose run --rm import-osm ./psql.sh  -P pager=off  -c "\d+"
 
 psql-pg-stat-reset:
-	docker-compose run --rm import-osm /usr/src/app/psql.sh  -P pager=off  -c 'SELECT pg_stat_statements_reset();'
+	docker-compose run --rm import-osm ./psql.sh  -P pager=off  -c 'SELECT pg_stat_statements_reset();'
 
 forced-clean-sql:
-	docker-compose run --rm import-osm /usr/src/app/psql.sh -c "DROP SCHEMA IF EXISTS public CASCADE ; CREATE SCHEMA IF NOT EXISTS public; "
-	docker-compose run --rm import-osm /usr/src/app/psql.sh -c "CREATE EXTENSION hstore; CREATE EXTENSION postgis; CREATE EXTENSION unaccent; CREATE EXTENSION fuzzystrmatch; CREATE EXTENSION osml10n; CREATE EXTENSION pg_stat_statements;"
-	docker-compose run --rm import-osm /usr/src/app/psql.sh -c "GRANT ALL ON SCHEMA public TO public;COMMENT ON SCHEMA public IS 'standard public schema';"
+	docker-compose run --rm import-osm ./psql.sh -c "DROP SCHEMA IF EXISTS public CASCADE ; CREATE SCHEMA IF NOT EXISTS public; "
+	docker-compose run --rm import-osm ./psql.sh -c "CREATE EXTENSION hstore; CREATE EXTENSION postgis; CREATE EXTENSION unaccent; CREATE EXTENSION fuzzystrmatch; CREATE EXTENSION osml10n; CREATE EXTENSION pg_stat_statements;"
+	docker-compose run --rm import-osm ./psql.sh -c "GRANT ALL ON SCHEMA public TO public;COMMENT ON SCHEMA public IS 'standard public schema';"
 
 pgclimb-list-views:
-	docker-compose run --rm import-osm /usr/src/app/pgclimb.sh -c "select schemaname,viewname from pg_views where schemaname='public' order by viewname;" csv
+	docker-compose run --rm import-osm ./pgclimb.sh -c "select schemaname,viewname from pg_views where schemaname='public' order by viewname;" csv
 
 pgclimb-list-tables:
-	docker-compose run --rm import-osm /usr/src/app/pgclimb.sh -c "select schemaname,tablename from pg_tables where schemaname='public' order by tablename;" csv
+	docker-compose run --rm import-osm ./pgclimb.sh -c "select schemaname,tablename from pg_tables where schemaname='public' order by tablename;" csv
 
 psql-vacuum-analyze:
 	@echo "Start - postgresql: VACUUM ANALYZE VERBOSE;"
-	docker-compose run --rm import-osm /usr/src/app/psql.sh  -P pager=off  -c 'VACUUM ANALYZE VERBOSE;'
+	docker-compose run --rm import-osm ./psql.sh  -P pager=off  -c 'VACUUM ANALYZE VERBOSE;'
 
 psql-analyze:
 	@echo "Start - postgresql: ANALYZE VERBOSE ;"
-	docker-compose run --rm import-osm /usr/src/app/psql.sh  -P pager=off  -c 'ANALYZE VERBOSE;'
+	docker-compose run --rm import-osm ./psql.sh  -P pager=off  -c 'ANALYZE VERBOSE;'
 
 list-docker-images:
 	docker images | grep openmaptiles

